staging/src/k8s.io/apiserver/.import-restrictions

```diff
@@ -1,10 +1,5 @@
-{
-  "Rules": [
-    {
-      "SelectorRegexp": "k8s[.]io/kubernetes",
-      "ForbiddenPrefixes": [
-        ""
-      ]
-    }
-  ]
-}
+rules:
+  # prevent import of k8s.io/kubernetes
+  - selectorRegexp: k8s[.]io/kubernetes
+    forbiddenPrefixes:
+      - ''
```

Comment:
preexisting, but is there a difference between `allowedPrefixes: []` (used in portforward) and `forbiddenPrefixes: [""]` (used here)?

Reply:
AFAICT they are effectively equivalent.  I think forbiddenPrefixes is newer.  I'm happy to pick one and standardize on it

Reply:
Not having `allowedPrefixes` present is the same as `allowedPrefixes: []`.  If there are no allowed prefixes, nothing's going to be allowed.  From that sense, the `forbiddenPrefixes` is redundant.

But, it does mean you also see `import k8s.io/kubernetes/pkg/foo has forbidden prefix` in import-boss' output, in addition to `the following imports did not match any allowed prefix: k8s.io/kubernetes/pkg/foo`

I don't have a strong opinion either way.

Reply:
I don't feel strongly either, just trying to understand what the different files are actually doing

Reply:
I opted to standardize on `forbiddenPrefixes: [""]` since it seems to describe intent a bit more